src/bun.js/webcore/response.zig

```diff
@@ -758,17 +758,7 @@ pub const Fetch = struct {
             return fetch_tasklet;
         }
 
-        const FetchOptions = struct {
-            method: Method,
-            headers: Headers,
-            body: AnyBlob,
-            timeout: usize,
-            disable_timeout: bool,
-            disable_keepalive: bool,
-            url: ZigURL,
-            verbose: bool = false,
-            follow_redirects: bool = true,
```

Comment:
btw

with zig fmt

the rule is

if there's a trailing comma, it gets printed on new lines

otherwise its all one line

Reply:
this is useful